doc/fence-config.md

```diff
@@ -0,0 +1,32 @@
+# Fence config helper
+Little helper to split/merge fence configs, or delete/extract/replace secrets from fence configs
+
+## Use
+
+* `gen3 fence-config extract-secrets [input-yaml-path] [output-json-path]`
```

Comment:
JSON because it's more human readable, and also keeping the yaml without losing comments are also for more human readable. I assume we need extra manual check from devops when splitting the secrets.

Reply:
json is more readable for devops to update